string gesture_name   # name of the gesture
uint8 max_poses       # numero of waypoints that the gesture has
float32[] poses       # array with all the poses
